website/docs/community/resources/code-of-conduct.md

```diff
@@ -5,46 +5,38 @@ id: "code-of-conduct"
-
-People genuinely love this community, and we are committed to maintaining the spirit of it. As such have written this Code of Conduct to help all participants understand how to best participate in our community.
-
-The Code of Conduct applies to all dbt Community spaces both online and off. This includes: Slack, Discourse, code repositories (dbt Core, dbt packages etc), Office Hours and Meetups. There are some guidelines specific to particular forums (listed below). Participants are responsible for knowing and abiding by this Code of Conduct.
-
-This Code of Conduct has three sections:
+This Code of Conduct has two sections:
 
 - **dbt Community Values:** These values apply to all of our community spaces, and all of our guidelines are based on these values.
-- **Forum-specific guidelines**: These guidelines explain some of the cultural norms that apply to specific forums.
 - **Anti-harassment policy:**  We are dedicated to providing a harassment-free experience for everyone in our community — here, we outline exactly what that means.
 
 We appreciate your support in continuing to build a community we’re all proud of.
 
 — The dbt Community Admin Team.
 
 ## dbt Community Values
+### Create more value than you capture.
 
-### Be respectful.
-
-We want everyone to have a fulfilling and positive experience in the dbt Community and we are continuously grateful in your help ensuring that this is the case.
-
-Be courteous, respectful, and polite to fellow community members. Generally, don’t be a jerk.
-
-Be considerate of others’ time — many people in the community generously give their time for free.
+Each community member should strive to create more value in the community than they capture. This is foundational to being a community.
 
 Ways to demonstrate this value:
 
-- Take the time to write bug reports well ([example](https://github.com/fishtown-analytics/dbt/issues/2370))
-- Thank people if they help solve a problem.
+- Contribute to dbt or a dbt package
+- Participate in discussions on Slack and the Community Forum
+- Share things you have learned on the Community Forum
+- Host events
+
+Be mindful that others may not want their image or name on social media, and when attending or hosting an in-person event, ask permission prior to posting about another person.
 
 ### Be you.
 
 Some developer communities allow and even encourage anonymity — we prefer it when people identify themselves clearly. It helps to build empathy, and form relationships.
 
 Ways to demonstrate this value:
 
-- Update your profile on any dbt Community forums to include your name, and a clear picture. On Slack, use the “what I do” section to add your role title and current company
+- Update your profile on any dbt Community platforms to include your name, and a clear picture. On Slack, use the "what I do" section to add your role title and current company
```

Comment:
I don't think clear picture is specific enough. I think we can suggest a picture of yourself if you're comfortable with that. My ever lasting fear is if someone represents themselves as someone else that they're not or a company they're not working at.

Reply:
Do you mean I should say "add a picture of yourself" if I am comfortable suggesting that? Or do you mean I should say "add a picture of yourself if you are comfortable doing so"?

I think it was the former and have changed accordingly